src/BizHawk.Emulation.Cores/Consoles/Sony/PSP/PPSSPP.IVideoProvider.cs

```diff
@@ -0,0 +1,48 @@
+using BizHawk.Emulation.Common;
+
+namespace BizHawk.Emulation.Cores.Consoles.Sony.PSP
+{
+	public partial class PPSSPP : IVideoProvider
+	{
+		static int BW = 480;
+		static int BH = 270;
+		internal bool VideoDirty;
+
+		protected readonly IntPtr _context;
+
+		public PPSSPP(LibPPSSPP core, IntPtr context)

```

Comment:
`core` argument is ignored? This constructor doesn't seem to be used anyway.

Reply:
removed constructor altogether